recipe/meta.yaml

```diff
@@ -5,14 +5,13 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ version }}.tar.gz
-  url: https://github.com/trungdong/prov/archive/{{ version }}.tar.gz
-  sha256: 499b46dfc96ed88dd7ca28ff0fdb89a7c9bd8e4956857efd3aa31fa97313a538
+  url: https://pypi.io/packages/source/p/prov/prov-{{ version }}.tar.gz
+  sha256: a0c951107eafd508d5a26caa54863dc11fee3da86778db2a559d32450d2030a9
```

Comment:
since version has not changed, why is the `sha` different?

Reply:
It might have been a push to the pypi repository by the author. Though the version has not change, the author can make an update to the package and pushed to pypi. I obtained this sha from pypi.

See: https://pypi.org/project/prov/1.5.3/#files
![screenshot from 2019-02-12 10-27-01](https://user-images.githubusercontent.com/17802172/52658809-c8d47780-2eb0-11e9-9a63-ff7d0ff51c8b.png)

It's on the sha256 link. Thanks.

Reply:
The first sha is from github, the second from pypi.

Reply:
@kmuehlbauer Thanks.